src/styles/_variable.scss

```
@@ -0,0 +1,12 @@
+$cell_size: 100px;
+$calendar_gap: 1px;
+$calendar_width: 6 * $calendar_gap + 7 * $cell_size;
+$cell_color: #eee;
+$start-day:
+  "mon" 0,
+  "tue" #{$cell-size} + #{$calendar-gap},
+  "wed" 2 * ($cell-size + $calendar-gap),
```

You can just calc it one time instead calc each time
```suggestion
$calendar_width: 6 * $calendar_gap + 7 * $cell_size;
$cell_color: #eee;
$day-margin: $cell-size + $calendar-gap;
$start-day:
  "mon" 0,
  "tue" $day-margin,
  "wed" 2 * $day-margin,
```